var/spack/repos/builtin/packages/cgal/package.py

```diff
@@ -16,6 +16,7 @@ class Cgal(CMakePackage):
     homepage = 'https://www.cgal.org/'
     url      = "https://github.com/CGAL/cgal/releases/download/releases/CGAL-5.0.3/CGAL-5.0.3.tar.xz"
```

Comment:
Can you also update the URL to the new syntax so that `spack versions cgal` and `spack checksum cgal` work?

Reply:
Great catch!  I'll have to be more careful next time.